dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -95,6 +95,9 @@
 )
 from airflow_breeze.utils.shared_options import get_dry_run, get_verbose, set_forced_answer
 from airflow_breeze.utils.visuals import ASCIIART, ASCIIART_STYLE, CHEATSHEET, CHEATSHEET_STYLE
+from docs.exts.docs_build.docs_builder import AirflowDocsBuilder
```

Comment:
Comment: The thing here is that whatever we have here in "docs.exts", it needs to be copied to "breeze". Breeze should be largely self-containing (except occasional retrieval of the information from "airflow" sources, importing the code should always be done as

```
from airflow_breeze.....
```

And the code imported should live in `dev/breeze/src/airflow_breeze` folder.

Reply:
(that's part of the problem to solve - to get rid of the "docs" folder that needs to be added on PYTHONPATH)

Reply:
Makes sense to me. Will re work this